WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -282,8 +282,8 @@ public void onLog(T tag, LogLevel logLevel, String message) {
         createNotificationChannelsOnSdk26();
 
         // Allows vector drawable from resources (in selectors for instance) on Android < 21 (can cause issues
-        // with memory usage and the use of Configuration). More informations: http://bit.ly/2H1KTQo
-        // Note: if removed, this will cause crashes on Android < 21
+        // with memory usage and the use of Configuration). More information: http://bit.ly/2H1KTQo
+        // Note: Android < 21 will crash if removed
```

Comment:
The original comment was perfectly fine too. I believe it was even clearer to understand.